addons/point_of_sale/static/src/js/printers.js

```diff
@@ -30,7 +30,7 @@ class PrintResultGenerator {
             successful: false,
             message: {
                 title: _t('Connection to the printer failed'),
-                body: _t('Please check if the printer is still connected. \n' +
+                body: _t('Please check if the printer is still connected and make sure there are no errors in the IoT box logs. \n' +
```

Comment:
Something else that you have to be careful of is the fact that this string is in a `_t(...)` (which is the python equivalent of `_()`.
Which means that this sentence is meant to be translated.

Here there is 2 issues with your change:
- You modify a string that might have existing translations (so they will be lost).
- You didn't update the .pot file with your new text

=> You will prevent the translation until someone update it for you & the previous translation will be lost

In this case you are a bit in trouble as the full line is translated as we can see in:
https://github.com/odoo/odoo/blob/5eeb565fb094ee4f51994bafceb2110760081228/addons/point_of_sale/i18n/point_of_sale.pot#L4124-L4126
But, what you could do is for instance appending at the end of the `_t(...)` your message. Like: 
```js
body: _t('Please check if the printer is still connected. \n' + 
...
'\'Self-signed certificate for ePOS printers\' and \'Secure connection (HTTPS)\' to solve the issue'
 ) 
+ "\n" 
+  _t("make sure there are no errors in the IoT box logs."),
```